### DISCUSSION TITLE
设置菜单层级关系

### DISCUSSION THREAD

**Discussion by FoundTheWOUT** [the1812/Bilibili-Evolved#2525] [提问 / Q&A] [answered]
在某些情况下, 感觉有点怪. 是故意为之吗
![image](https://user-images.githubusercontent.com/47922588/141798694-061d1d03-78bc-4f33-bf6b-e20f113bbefa.png)

**Comment by the1812** [comment, marked as answer]
是 bug, 顺便图透一下下个版本的, 这种选项在设置里会直接两列铺开, 在功能面板里才弹窗
![image](https://user-images.githubusercontent.com/26504152/141800530-b553590d-b271-4877-ac1f-7aa05ee0e700.png)

**Reply by kethoven** [reply]
这个变化我喜欢。
更加方便直观。
